torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -203,7 +203,10 @@ def is_dequantize_node(node):
 #   1) The inner reference module class
 #   2) The replacement quantized module class for lowering
 LOWER_FUSED_MODULE_MAP: Dict[Type[nn.Module], Tuple[Type[nn.Module], Type[WeightedQuantizedModule]]] = {
-    nni.LinearReLU: (nnqr.Linear, nniq.LinearReLU)
+    nni.LinearReLU: (nnqr.Linear, nniq.LinearReLU),
+    nni.ConvReLU1d: (nnqr.Conv1d, nniq.ConvReLU1d),
+    nni.ConvReLU2d: (nnqr.Conv2d, nniq.ConvReLU2d),
+    nni.ConvReLU3d: (nnqr.Conv3d, nniq.ConvReLU3d),
```

Comment:
This is just conv-relu so far right? Conv-bn will be added later?

Reply:
We won't have conv bn in the final reference quantized model I think, bn will be fused into conv and we are left with ConvRelu only

Reply:
sorry, I think you are right, we will still have conv-bn-relu module in the end, it should be fused in the `from_reference` call

Reply:
I see. Well I think even for conv-bn we still need to add something like this to the map:

Reply:
actually I took a look again, and it looks like we need to fuse bn into conv/linear before we observe the weights, so it needs to happen in quantization_patterns.py instead of the lowering